src/lib/date.ts

```diff
@@ -1,8 +1,32 @@
 import moment from "moment"
 
+const formattedDateStatusDate = date => {
+  const momentToUse = moment.utc(date, "YYYY-MM-DD[T]HH:mm:ss")
+  const momentDate = momentToUse.format("MMM D")
+  const momentHour = momentToUse.format("ha")
```

Comment:
This is another case of dropping minute precision right? I'm not sure we should ever use `ha` as a display format, since that is by definition lossy with respect to minutes.